src/frontend/src/optimizer/rule/table_function_to_internal_backfill_progress.rs

```diff
@@ -65,6 +65,7 @@ impl TableFunctionToInternalBackfillProgressRule {
                 Field::new("fragment_id", DataType::Int32),
                 Field::new("backfill_state_table_id", DataType::Int32),
                 Field::new("current_row_count", DataType::Int64),
+                Field::new("snapshot_backfill_epoch", DataType::Int64),
```

Comment:
Should it be `min_epoch`? `snapshot_backfill_epoch` can be misunderstood as the epoch of the snapshot we consume on.